### PR TITLE
WIP [do not merge] adds graceful shutdown vars for testing

### PR DESCRIPTION
**Pull request by wgahnagl**
**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by sinnykumari** [comment]
You may need to apply https://github.com/openshift/machine-config-operator/pull/2509 as well for testing this feature.

File: templates/master/01-master-kubelet/_base/files/kubelet.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ contents:
     kubeAPIQPS: 50
     kubeAPIBurst: 100
     rotateCertificates: true
+    shutdownGracePeriod: 100
```
Comment (rphillips) [inline templates/master/01-master-kubelet/_base/files/kubelet.yaml:22]: these times will need to be much larger... maybe 600 for 10 minutes.

**Comment by rphillips** [comment]
Adding a breadcrumb to this PR.... We ran into issues with the Graceful Shutdown feature, and it will need to be fixed upstream.

**Pull request closed without merging**